amalgamation/Makefile

```diff
@@ -16,6 +16,9 @@ else
 	DEFS+=-DMSHADOW_USE_SSE=0
 endif
 
+ifndef APPLE
+    export APPLE=0
```

Comment:
@liangfu I checked the latest version on this branch, and the Makefile doesn't seem to depend on this flag any more. If it's not used, could you make sure it's removed? Thanks.